tests/tt_eager/python_api_testing/non_working_unit_tests/grayskull/test_argmax.py

```diff
@@ -39,7 +40,7 @@ def test_argmax(self, input_shapes, dim, all, device, enable_async, num_loops):
                 .to(tt_lib.tensor.Layout.TILE)
                 .to(device)
             )
-            tt_output_tensor_on_device = tt_lib.tensor.argmax(input_tensor, dim=dim, all=all)
+            tt_output_tensor_on_device = ttnn.experimental.argmax(input_tensor, dim=dim, all=all)
```

Comment:
Why not ttnn.argmax?

Reply:
As we have the op in experimental structure and ckernel implementation, so to show some difference its called as ttnn.experimental.argmax